src/utils/const.ts

```diff
@@ -68,6 +68,19 @@ export enum ClusterStatus {
   STATUS_STOPPED = 'STOPPED',
   STATUS_ACTIVE = 'ACTIVE'
 }
+export const ClusterStatusState: any = {
+  '0': 'UNKNOWN',
+  '1': 'CREATING',
+  '2': 'RUNNING',
+  '3': 'ERROR',
+  '4': 'DELETING',
+  '5': 'UPDATING',
+  '6': 'STOPPING',
+  '7': 'STOPPED',
+  '8': 'STARTING',
+  '9': 'ERROR_DUE_TO_UPDATE',
+  '10': 'REPAIRING'
```

Comment:
This is reproducing a lot of constant strings that are already defined below here (everything except `UNKNOWN`, `UPDATING`, `ERROR_DUE_TO_UPDATE`, and `REPAIRING`).

Instead, move this dictionary to the end of the file, add new constants for the 4 entries that don't already have one, and then reference the existing constants rather than reproducing them.